.tool/pin-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+cd "$(dirname $0)/.."
+
+if [ ! -x "$(command -v curl )" ] || [ ! -x "$(command -v jq )" ] ||  [ ! -x "$(command -v find )" ] ||  [ ! -x "$(command -v sed )" ]; then 
```

Comment:
You're going to change the list of commands, but in general, the `command` built-in is pretty well behaved so this can be somewhat simpler/cleaner too:

```suggestion
if ! command -v curl > /dev/null || ! command -v jq > /dev/null || ! command -v find > /dev/null || ! command -v sed > /dev/null; then
```

or even:

```suggestion
if ! { command -v curl && command -v jq && command -v find && command -v sed; } > /dev/null; then
```